lib/views/after_auth_screens/profile/profile_page.dart

```diff
@@ -6,20 +6,25 @@ import 'package:talawa/constants/routing_constants.dart';
 import 'package:talawa/enums/enums.dart';
 import 'package:talawa/locator.dart';
 import 'package:talawa/models/options/options.dart';
+import 'package:talawa/models/organization/org_info.dart';
+import 'package:talawa/models/post/post_model.dart';
+import 'package:talawa/models/user/user_info.dart';
 import 'package:talawa/plugins/talawa_plugin_provider.dart';
 import 'package:talawa/services/size_config.dart';
 import 'package:talawa/utils/app_localization.dart';
 import 'package:talawa/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart';
 import 'package:talawa/view_model/main_screen_view_model.dart';
+import 'package:talawa/views/after_auth_screens/feed/individual_post.dart';
 import 'package:talawa/views/base_view.dart';
 import 'package:talawa/widgets/custom_avatar.dart';
 import 'package:talawa/widgets/custom_list_tile.dart';
 import 'package:talawa/widgets/from_palisadoes.dart';
 import 'package:talawa/widgets/raised_round_edge_button.dart';
 
 /// ProfilePage returns a widget that renders a page of user's profile.
+// ignore: must_be_immutable
 class ProfilePage extends StatelessWidget {
-  const ProfilePage({
+  ProfilePage({
```

Comment:
Remove dummy data as suggested and mark this class as const again. Having const classes improves performance; we should not remove them unnecessarily.